miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/SampleAliquotSingleCellDto.java

```diff
@@ -0,0 +1,22 @@
+package uk.ac.bbsrc.tgac.miso.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+import uk.ac.bbsrc.tgac.miso.core.data.SampleAliquotSingleCell;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonTypeName(value = SampleAliquotSingleCell.SAMPLE_CLASS_NAME)
+public class SampleAliquotSingleCellDto extends SampleAliquotDto {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This class has 8 parents which is greater than 5 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMaximumInheritanceDepth)